build/ci/phase-template.yml

```diff
@@ -34,9 +34,18 @@ phases:
       displayName: Download Benchmark Data
     - ${{ if eq(parameters.name, 'MacOS') }}:
       - script: brew update && brew install libomp mono-libgdiplus gettext && brew link gettext --force
-        displayName: Install runtime dependencies
+        displayName: Install runtime dependencies     
+    - script: $(_buildScript) -$(_configuration) -runtests -coverage=true
+      displayName: Run Tests with code coverage.
+      condition: and(eq(variables._phaseName, 'Windows_x64'), eq(variables._configuration, 'Debug'))
```

Comment:
Do we really want to run code coverage in all PRs in the required build? Wouldn't be better to have an extra PR leg?

Reply:
Myself and @sharwell decided against it after making pref improvements to code coverage. By creating an extra leg you are also wasting resources. Code coverage just runs on one leg of the required build and it does not impact the merge, i.e if you regress code coverage it won't block.

Reply:
Sounds good! Btw, you could simplify that condition if you declared a variable in your matrix, but it is just a small not important thing I won't block the PR on.